### PR TITLE
[PERFSCALE-2435] Add Control Plane Tests - HyperShift

### PR DESCRIPTION
**Pull request by krishvoor**
This PR adds control-plane tests for
medium & large scale tests in 4.13 & 4.14 HyperShift Versions

### CODE REVIEW

**Comment by krishvoor** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-perfscale-rosa-hcp-multiaz-control-plane-120nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml [hunk 1]
```diff
@@ -253,6 +253,40 @@ tests:
     - chain: openshift-qe-hypershift-infra
     - chain: openshift-qe-data-path-tests
     workflow: rosa-aws-sts-hypershift-mp
+- always_run: false
+  as: perfscale-rosa-hcp-multiaz-control-plane-120nodes
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml:262]: m5.xlarge

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml [hunk 2]
```diff
@@ -253,6 +253,40 @@ tests:
...
+    test:
+    - chain: openshift-qe-ocp-qe-perfscale-ci-tests
+    workflow: rosa-aws-sts-hypershift
+- always_run: false
+  as: perfscale-rosa-hcp-multiaz-control-plane-252nodes
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml:279]: m5.xlarge

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml [hunk 3]
```diff
@@ -36,6 +36,40 @@ tests:
     - chain: openshift-qe-ocp-qe-perfscale-ci-tests
     workflow: rosa-aws-sts
   timeout: 6h0m0s
+- always_run: false
+  as: perfscale-rosa-hcp-multiaz-control-plane-120nodes
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml:45]: m5.xlarge

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml [hunk 4]
```diff
@@ -36,6 +36,40 @@ tests:
...
+    test:
+    - chain: openshift-qe-ocp-qe-perfscale-ci-tests
+    workflow: rosa-aws-sts-hypershift
+- always_run: false
+  as: perfscale-rosa-hcp-multiaz-control-plane-252nodes
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-nightly.yaml:62]: m5.xlarge

**Review by jtaleric — CHANGES_REQUESTED** [review]
compute type should be `m5.xlarge`

**Comment by krishvoor** [comment]
> compute type should be `m5.xlarge`

updated- ptal!

**Comment by krishvoor** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-x86-perfscale-rosa-hcp-multiaz-control-plane-120nodes

**Comment by krishvoor** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-x86-perfscale-rosa-multiaz-cluster-density-v2-3nodes

**Pull request closed without merging**